vips/vip-400/bscmainnet.ts

```diff
@@ -23,37 +20,108 @@ import {
   incentiveAndAccessibilities,
 } from "../../multisig/proposals/arbitrumone/vip-018/commands";
 
-const { arbitrumone } = NETWORK_ADDRESSES;
-
-const { NORMAL_TIMELOCK, FAST_TRACK_TIMELOCK, CRITICAL_TIMELOCK } = arbitrumone;
-
 export const ARBITRUM_COMPTROLLER_CORE = "0x317c1A5739F39046E20b08ac9BeEa3f10fD43326";
 export const ARBITRUM_COMPTROLLER_LST = "0x52bAB1aF7Ff770551BD05b9FC2329a0Bf5E23F16";
 export const ARBITRUM_PRIME = "0xFE69720424C954A2da05648a0FAC84f9bf11Ef49";
 export const ARBITRUM_PLP = "0x86bf21dB200f29F21253080942Be8af61046Ec29";
 
-export const ARBITRUM_USDT = "0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9";
-export const ARBITRUM_USDC = "0xaf88d065e77c8cc2239327c5edb3a432268e5831";
-export const ARBITRUM_WBTC = "0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f";
-export const ARBITRUM_WETH = "0x82af49447d8a07e3bd95bd0d56f35241523fbab1";
+export const ARBITRUM_USDT = "0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9";
+export const ARBITRUM_USDC = "0xaf88d065e77c8cC2239327C5EDb3A432268e5831";
+export const ARBITRUM_WBTC = "0x2f2a2543B76A4166549F7aaB2e75Bef0aefC5B0f";
+export const ARBITRUM_WETH = "0x82aF49447D8a07e3bd95BD0d56f35241523fBab1";
```

Comment:
Using checksummed addresses, nothing else

Reply:
done, https://github.com/VenusProtocol/vips/pull/429/commits/3c7166c46cbcf45ca2b0585fcef09e1b3b11fbc4

Reply:
The addresses were already checksummed. We should revert https://github.com/VenusProtocol/vips/commit/3c7166c46cbcf45ca2b0585fcef09e1b3b11fbc4

Reply:
done.